.github/ISSUE_TEMPLATE/bug-report-playback.yaml

```
@@ -37,46 +68,61 @@ body:
         Please paste your FFmpeg logs here. You can find these in your servers dashboard under "logs".
       placeholder: Paste logs…
       render: shell
-  - type: textarea
-    id: mediainfo
+  - type: markdown
     attributes:
-      label: Media info of the file
-      description: |
-        Please share the media information for the file causing issues. You can use a variety of tools to retrieve this information.
-        - Use ffprobe (`ffprobe ./file.mp4`)
-        - Use the Media Info tool (set to text format, download here: https://mediaarea.net/en/MediaInfo)
-        - Copy the media info from the web interface
-      placeholder: Paste media info…
-      render: shell
+      value: |
+        ## Software versions
```

Let's be fancy and call this

```suggestion
        ## Environment
```